### PR TITLE
verify excerpt validity

### PR DESCRIPTION
**Pull request by ibrahni**

### CODE REVIEW

File: src/test/java/com/baeldung/selenium/common/ArticlesUITest.java [hunk 1]
```diff
@@ -165,6 +165,34 @@ public final void givenAllArticles_whenAnalyzingImages_thenImagesDoNotHaveEmptyA
 
     }
 
+    @Test
+    public void givenAllArticles_whenAnalyzingExcerpt_thenItShouldNotBeEmptyAndShouldMuchDescription() {
```
Comment (asif-anwar) [inline src/test/java/com/baeldung/selenium/common/ArticlesUITest.java:169]: I think we have a typo in the test name (notice the bold part below)

givenAllArticles_whenAnalyzingExcerpt_thenItShouldNotBeEmptyAndShould**Much**Description

File: src/test/java/com/baeldung/selenium/common/ArticlesUITest.java [hunk 2]
```diff
@@ -165,6 +165,34 @@ public final void givenAllArticles_whenAnalyzingImages_thenImagesDoNotHaveEmptyA
...
+            recordExecution(GlobalConstants.givenAllArticles_whenAnalyzingExcerpt_thenItShouldNotBeEmptyAndShouldMuchDescription);
+
+            if (shouldSkipUrl(GlobalConstants.givenAllArticles_whenAnalyzingExcerpt_thenItShouldNotBeEmptyAndShouldMuchDescription)) {
+                continue;
+            }
+
+            final String metaDescriptionTag = page.getMetaDescriptionContent();
+            final String excerptTag = page.getMetaExcerptContent();
+
+            if (StringUtils.isBlank(excerptTag) || Objects.equals(excerptTag, metaDescriptionTag)) {
```
[the remark this answers is not in the document]
Comment (ibrahni) [reply src/test/java/com/baeldung/selenium/common/ArticlesUITest.java]: done, yes you are right I missed a !

File: src/test/java/com/baeldung/selenium/common/ArticlesUITest.java (hunk 1 above)
Comment (ibrahni) [reply src/test/java/com/baeldung/selenium/common/ArticlesUITest.java]: corrected

**Comment by ibrahni** [comment]
@asif-anwar 
changes done

**Review by asif-anwar — APPROVED** [review]

**Pull request merged**